src/react-components/avatar-preview.js

```diff
@@ -63,6 +63,26 @@ function fitBoxInFrustum(camera, box, center, margin = DEFAULT_MARGIN) {
   camera.lookAt(center);
 }
 
+function getThemeBackground() {
+  const currentTheme = APP.store.state.preferences.theme;
+  const themes = window.APP_CONFIG?.theme.themes;
+  const defaultColor = 0xeaeaea;
+  if (currentTheme === "hubs-default") {
+    return defaultColor;
+  }
+  for (let i = 0; i < themes.length; i++) {
+    if (themes[i].id === currentTheme) {
+      let bgHex = themes[i].variables["background3-color"];
+      bgHex = `0x${bgHex.substring(1)}`;
+      if (bgHex.length !== 8) {
+        return defaultColor;
+      }
+      bgHex = parseInt(bgHex, 16);
+      return bgHex;
+    }
+  }
```

Comment:
In case the `currentTheme` was not found in the themes array, return the defaultColor here.

This is currently a bug if you look at the behavior of the code in light mode when using `npm run dev`, since the default theme preference is undefined. The theme preference could also be "Browser Default", which is arguably a bug in the preferences panel.

```suggestion
  }
  
  return defaultColor;
```